src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -926,19 +925,19 @@ async function applyFeatureConfigToFeature(output: Log, featureSet: FeatureSet,
 	const jsonString: Buffer = await readLocalFile(innerJsonPath);
 	const featureJson = jsonc.parse(jsonString.toString());
 
-	// TODO: Use spread operator {..., } to avoid needing to explicitly set each property.
 
-	feature.containerEnv = featureJson.containerEnv;
-	feature.buildArg = featureJson.buildArg;
-	feature.options = featureJson.options;
```

Comment:
I feel like we should never been assigning anything to `featureJson.options` here, as that is kind of a special case in the `devcontainer-feature.json` as UI metadata (https://github.com/devcontainers/features/blob/main/src/docker-from-docker/devcontainer-feature.json#L6-L31).

I think this is just ignored later on, and the options are passed along separately.